src/server/scripts/Kalimdor/boss_azuregos.cpp

```diff
@@ -83,7 +79,6 @@ class boss_azuregos : public CreatureScript
         {
             if (victim && victim->GetTypeId() == TYPEID_PLAYER)
             {
-                Talk(SAY_KILL);
```

Comment:
Don't remove this. The issue must be somewhere else, it doesn't make any sense since this only uses the owner of the creature text.

Reply:
Returned